src/stringProcessing/syllables/count.js

```diff
@@ -172,7 +172,6 @@ var countSyllablesInText = function( text, locale ) {
 	var syllableCounts = map( words,  function( word ) {
 		return countSyllablesInWord( word, locale );
 	} );
-
```

Comment:
The empty line before the return statement should be restored (I guess this might have been removed by mistake)